sematic/scheduling/kubernetes.py

```diff
@@ -891,3 +894,35 @@ def _shared_memory() -> Tuple[V1Volume, V1VolumeMount]:
     volume_mount = V1VolumeMount(mount_path="/dev/shm", name=volume_name)
 
     return volume, volume_mount
+
+
+def _get_image_pull_secrets() -> Optional[
```

Comment:
Please add a docstring stating that this must return `[list[V1LocalObjectReference]](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1LocalObjectReference.md)` (with the link).

Reply:
done